Composer/plugins/azurePublish/src/index.ts

```diff
@@ -195,11 +195,16 @@ class AzurePublisher {
       }
     } catch (error) {
       console.log(error);
+      if (typeof error === 'object') {
+        this.logMessages.push(JSON.stringify(error));
+      } else {
+        this.logMessages.push(error);
+      }
       // update status and history
       const status = this.getLoadingStatus(botId, profileName, jobId);
       if (status) {
         status.status = 500;
-        status.result.message = error ? error.message : 'publish error';
+        status.result.message = error && error.message ? error.message : 'publish error';
```

Comment:
We can use the new null coalescing to write this more cleanly:
`status.result.message = error?.message ?? 'publish error'`

Reply:
done